shenyu-plugin/shenyu-plugin-brpc/src/main/java/org/apache/shenyu/plugin/brpc/cache/ApplicationConfigCache.java

```diff
@@ -152,26 +173,44 @@ public AsyncGenericService build(final MetaData metaData) {
      */
     public AsyncGenericService get(final String path) {
         try {
-            return cache.get(path);
+            return serviceCache.get(path);
         } catch (ExecutionException e) {
             throw new ShenyuBrpcPluginException(e.getCause());
         }
     }
 
+    /**
+     * init brpc service config.
+     *
+     * @param metaData the meta data
+     * @return service config
+     */
+    public ServiceConfig initServiceConfig(final MetaData metaData) {
```

Comment:
In `org.apache.shenyu.plugin.brpc.handler.BrpcMetaDataHandler#handle`, maybe just change `org.apache.shenyu.plugin.brpc.cache.ApplicationConfigCache#initService` to `org.apache.shenyu.plugin.brpc.cache.ApplicationConfigCache#initServiceConfig` ?